src/main/apiClient.ts

```diff
@@ -141,9 +142,8 @@ export default class ApiClient {
         body: {
           clientId: this.config.clientId,
           grantType: 'authorization_code',
-          code: params.code,
-          redirectUri: params.redirectUri,
-          code_verifier: sessionStorage.getItem('verifier_key')
+          code_verifier: sessionStorage.getItem('verifier_key'),
```

Comment:
This variable can be in camel case.

```suggestion
          codeVerifier: sessionStorage.getItem('verifier_key'),
```

Reply:
I'm sorry you can't because it's the shape of expected body payload in endpoint `/oauth/token`
```private def tokenFromAuthorizationCode(
    client: Client,
  )(implicit req: RequestWithDomain[_],
  ): Future[Either[AppError, Map[String, Any]]] =
    trace("IdentityApi.tokenFromAuthorizationCode") {
      validateForm {
        Form(
          tuple(
            "code"          -> nonEmptyText,
            "redirect_uri"  -> nonEmptyText,
            "code_verifier" -> optional(nonEmptyText(43, 128)),
          ),
        )
      }.traverse {
        case (code, redirectUri, codeVerifier) =>
          authService.validateAuthCode(code, redirectUri, client, codeVerifier)
      }
    }```

Reply:
The SDK Core transforms all the requests parameters into snake case https://github.com/ReachFive/identity-web-core-sdk/blob/master/src/main/httpClient.ts#L67
So it should be ok to convert the variable into camel case :)

Reply:
Alright, actually tested it and the payload indeed contains the snake_cased version of the param. Updated in the code.